openshift-knative-operator/pkg/common/api.go

```diff
@@ -0,0 +1,177 @@
+package common
+
+import (
+	"fmt"
+
+	"strings"
+
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	"github.com/blang/semver/v4"
+	mf "github.com/manifestival/manifestival"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"knative.dev/pkg/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+// UpgradePodDisruptionBudget upgrade the API version to policy/v1
+func UpgradePodDisruptionBudget() mf.Transformer {
```

Comment:
Fine having this here, but isn't this also somewhat a concern for the upstream operator?

Reply:
The upstream operator has more freedom on what it supports (I guess it will make sure that it works on the three latest K8s versions). We have to support more versions.